src/screens/earn/farm-item/farm-item.tsx

```diff
@@ -73,24 +74,29 @@ export const FarmItem: FC<Props> = ({ farm, lastStakeRecord }) => {
   );
 
   const apr = useMemo(
-    () => (isDefined(farm.item.apr) ? new BigNumber(farm.item.apr).toFixed(DEFAULT_DECIMALS) : '---'),
+    () => (isDefined(farm.item.apr) ? aprToApy(Number(farm.item.apr)).toFixed(DEFAULT_DECIMALS) : '---'),
     [farm.item.apr]
   );
-  const depositAmountAtomic = useMemo(
-    () =>
-      mutezToTz(new BigNumber(lastStakeRecord?.depositAmountAtomic ?? DEFAULT_AMOUNT), FARM_PRECISION).multipliedBy(
-        farm.item.depositExchangeRate ?? DEFAULT_EXHANGE_RATE
-      ),
-    [lastStakeRecord?.depositAmountAtomic]
-  );
-  const claimableRewardsAtomic = useMemo(
-    () =>
-      mutezToTz(
-        new BigNumber(lastStakeRecord?.claimableRewards ?? DEFAULT_AMOUNT),
-        farm.item.rewardToken.metadata.decimals
-      ).multipliedBy(farm.item.earnExchangeRate ?? DEFAULT_EXHANGE_RATE),
-    [lastStakeRecord?.claimableRewards]
-  );
+
+  const depositAmountAtomic = useMemo(() => {
+    if (isDefined(lastStakeRecord) && isDefined(lastStakeRecord.depositAmountAtomic)) {
+      return mutezToTz(new BigNumber(lastStakeRecord.depositAmountAtomic), FARM_PRECISION)
+        .multipliedBy(farm.item.depositExchangeRate ?? DEFAULT_EXHANGE_RATE)
+        .toFixed(DEFAULT_DECIMALS);
+    }
+
+    return DEFAULT_AMOUNT;
+  }, [lastStakeRecord?.depositAmountAtomic]);
+
+  const claimableRewardsAtomic = useMemo(() => {
+    if (isDefined(lastStakeRecord) && isDefined(lastStakeRecord.claimableRewards)) {
```

Comment:
```suggestion
    if (isDefined(lastStakeRecord?.claimableRewards)) {
```

Reply:
The same

Reply:
Attention: make sure that tsc doesn't produce errors because of these two suggestions